tests/lib/external/snapd-testing-tools/tools/tests.pkgs.dnf-yum.sh

```diff
@@ -73,3 +73,12 @@ cmd_remove() {
         yum remove -y $@
     fi
 }
+
+cmd_download() {
+    # shellcheck disable=SC2068
+    if [ "$(command -v dnf)" != "" ]; then
+        dnf download $@ >/dev/null 2>&1
```

Comment:
```suggestion
        dnf download "$@" >/dev/null 2>&1
```

Is the output from `dnf` to overbose? Could we pass --quiet?

Reply:
updated